solvebio/resource/task.py

```diff
@@ -27,3 +27,15 @@ def child_object(self):
     def follow(self):
         """ Follow the child object but do not loop """
         self.child_object.follow(loop=False)
+
+    def cancel(self):
```

Comment:
might actually be cleaner to make Task and UpdateableResource and send the update there.